nise/report.py

```diff
@@ -1168,22 +1168,19 @@ def oci_bucket_upload(bucket_name, report_type, month, year, data, options):
 
 def oci_create_report(options):
     """Create cost and usage report files."""
+
     start_date = options.get("start_date")
     end_date = options.get("end_date")
-    fake = Faker()
-    attributes = {}
-    attributes["tenant_id"] = f"ocid1.tenancy.oc1..{fake.pystr(min_chars=20, max_chars=30)}"
-    attributes["compartment_name"] = fake.name().replace(" ", "").lower()
     static_report_data = options.get("static_report_data")
 
     if static_report_data:
         generators = _get_generators(static_report_data.get("generators"))
     else:
         generators = [
-            {"generator": OCIComputeGenerator, "attributes": attributes},
-            {"generator": OCIBlockStorageGenerator, "attributes": attributes},
-            {"generator": OCINetworkGenerator, "attributes": attributes},
-            {"generator": OCIDatabaseGenerator, "attributes": attributes},
+            {"generator": OCIComputeGenerator, "attributes": {}},
```

Comment:
And then we don't have to set it to empty dictionaries here.

Reply:
👍🏿  thanks